src/components/section/ohri-form-section.component.tsx

```diff
@@ -38,30 +47,85 @@ const OHRIFormSection = ({ fields, onFieldChange }) => {
+                ? previousValueDisplayForCheckbox(prevValue)
+                : null;
+
               return (
-                <div key={index} className={styles.parent}>
-                  {qnFragment}
+                <div key={index} className={styles.parentResizer}>
+                  <div
+                    className={
+                      fieldDescriptor.questionInfo &&
+                      `${
+                        fieldDescriptor.questionOptions.rendering !== 'radio'
+                          ? styles.questionInfoCentralized
+                          : styles.questionInfoDefault
+                      }
+                      `
+                    }>
+                    <div
+                      className={`${
+                        fieldDescriptor.questionOptions.rendering == 'radio' ||
+                        fieldDescriptor.questionOptions.rendering == 'date' ||
+                        fieldDescriptor.questionOptions.rendering == 'datetime'
+                          ? ''
+                          : styles.flexBasisOn
+                      } ${fieldDescriptor.setWidthConstraint && styles.controlWidthConstrained}`}>
+                      {qnFragment}
+                    </div>
+                    {fieldDescriptor.questionInfo && (
+                      <div className={styles.questionInfoControl}>
+                        {' '}
+                        <OHRITooltip field={fieldDescriptor} />{' '}
+                      </div>
+                    )}
+                  </div>
+
                   <div
                     className={
-                      isUnspecifiedSupported(fieldDescriptor) ? styles.tooltipWithUnspecified : styles.tooltip
+                      isUnspecifiedSupported(fieldDescriptor)
+                        ? styles.unspecifiedQuestionInfoSpaceBetween
+                        : styles.questionInfoFlexEnd
                     }>
                     {isUnspecifiedSupported(fieldDescriptor) &&
                       fieldDescriptor.questionOptions.rendering != 'group' && (
                         <OHRIUnspecified question={fieldDescriptor} onChange={onFieldChange} handler={handler} />
                       )}
-                    {fieldDescriptor.questionInfo && <OHRITooltip field={fieldDescriptor} />}
                   </div>
+                  {encounterContext?.previousEncounter &&
+                    prevValue &&
+                    !isTrue(fieldDescriptor.questionOptions.usePreviousValueDisabled) && (
+                      <div className={styles.previousValue}>
+                        <PreviousValueReview
+                          value={prevValue}
+                          displayText={
+                            fieldDescriptor.questionOptions.rendering == 'checkbox'
+                              ? prevCheckboxDisplayValue
+                              : prevValue?.display
+                          }
```

Comment:
Pay attention to the usages of `previousValue` and `prevValue` when aligning the naming.